tests/v2/validation/provisioning/k3s/README.md

```diff
@@ -299,6 +300,26 @@ These tests utilize Go build tags. Due to this, see the below examples on how to
 
 If the specified test passes immediately without warning, try adding the `-count=1` flag to get around this issue. This will avoid previous results from interfering with the new test run.
 
+## Template Test
+
+Dependencies:
+* [Cloud Credential](#cloud-credentials)
+* Make sure the template provider matches the credentials.
+```yaml
+templateTest:
+  repo:
+    metadata:
+      name: "sgtest"
+    spec:
+      gitRepo: https://github.com/susesgartner/cluster-template-examples.git
```

Comment:
nit - let's just have your forked repo removed to something like `https://github.com/<forked repo>/cluster-template-examples.git`.